internal/cmdrender/executor.go

```diff
@@ -193,14 +211,19 @@ type pkgNode struct {
 }
 
 // newPkgNode returns a pkgNode instance given a path or pkg.
-func newPkgNode(path string, p *pkg.Pkg) (pn *pkgNode, err error) {
+func newPkgNode(path string, p *pkg.Pkg, fs filesys.FileSystem) (pn *pkgNode, err error) {
 	const op errors.Op = "pkg.read"
 
 	if path == "" && p == nil {
 		return pn, fmt.Errorf("missing package path %s or package", path)
 	}
 	if path != "" {
-		p, err = pkg.New(path)
+		if fs == nil {
+			p, err = pkg.New(path)
```

Comment:
We could use the kyaml abstraction of FileSysemOrOnDisk; not feeling strongly about it but if the branching becomes unwieldy, it could be an option.

Reply:
Removed branching.